xain_fl/grpc/participant.py

```diff
@@ -83,6 +83,8 @@ def end_training(
         history (obj:`History`): History metadata.
         Metrics (obj:`Metrics`): Metrics metadata.
     """
+    # TODO:
+    # pylint: disable=no-member
```

Comment:
Can you explain this TODO please?

Reply:
Honestly, there is no reason for it. In the meantime I got the information, that we shouldn't change the participant grpc code in the xain fl package. I'll remove it.